test/system/README.md

```diff
@@ -46,13 +46,14 @@ without having to wait for the entire test suite.
 
 Running tests
 =============
-To run the tests locally in your sandbox, you can use one of these methods:
-* make;PODMAN=./bin/podman bats ./test/system/070-build.bats # runs just the specified test
-* make;PODMAN=./bin/podman bats ./test/system                # runs all
-* make;PODMAN=./bin/podman NETWORK_BACKEND=netavark bats ./test/system  # Assert & enable netavark testing
+To run the tests locally in your sandbox using `hack/bats` is recommend, check `hack/bats --help` for info about usage.
 
-To test as root:
-*  $ PODMAN=./bin/podman sudo --preserve-env=PODMAN bats test/system
+Alternatively you can use:
+
+* PODMAN=./bin/podman bats ./test/system/070-build.bats # runs just the specified test
+* PODMAN=./bin/podman bats ./test/system                # runs all
```

Comment:
This doesn't actually work: several system tests `cd` out of the current directory. And this will not set the `HELPER` dir, so quadlet and rootlessport will use the system-installed version. I cannot ever recommend that anyone ever do this.

Reply:
Yes, sorry for not being clear about that. The existing mention of `hack/bats` and `make {local,remote}system` suffice.

Reply:
done